src/App.tsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setSearch] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    return movie.title.toLowerCase().includes(query.toLowerCase().trim())
+      || movie.description.toLowerCase().includes(query.toLowerCase().trim());
```

Comment:
A lot of repeated logic.
Now it is not readable.

```suggestion
    const movieTitle = movie.title.toLowerCase();
    const movieDesc = movie.description.toLowerCase();
    const normilizedQuery = query.toLowerCase().trim();
    
    return movieTitle.includes(normilizedQuery)
     || movieDesc.includes(normilizedQuery);
```